src/Drivers/Gemini/Client.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace EchoLabs\Prism\Drivers\Gemini;
+
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Http;
+
+class Client
+{
+    private readonly PendingRequest $client;
+
+    private const string BASE_URL = 'https://generativelanguage.googleapis.com/v1beta';
```

Comment:
don't you think this should be config value ?

Reply:
Yep, you're right - I'll update.